public/app/features/explore/extensions/getExploreExtensionConfigs.tsx

```diff
@@ -38,6 +41,23 @@ export function getExploreExtensionConfigs(): PluginExtensionLinkConfig[] {
           });
         },
       }),
+      createExtensionLinkConfig<PluginExtensionExploreContext>({
+        title: 'Add Correlation',
+        description: 'Create a correlation from this query',
+        extensionPointId: PluginExtensionPoints.ExploreToolbarAction,
+        icon: 'link',
+        configure: (context) => {
+          const shouldShow =
+            context?.shouldShowAddCorrelation !== undefined ? context?.shouldShowAddCorrelation : false;
+          return shouldShow ? {} : undefined;
```

Comment:
Is it the same as below?

```suggestion
          return context?.shouldShowAddCorrelation ? {} : undefined;
```

Reply:
I thought I got paid by the word. Fixed :p

Reply:
> I thought I got paid by the word. Fixed :p

Lol, wish that was true. I would call my var `shouldItReallyReallyReallyReallyReallyReallyReallyReallyShowCorrelationOrMaybeNotWhatDoYouThinkImNotSureButLetsTryThisOneAndShowWhenTheContextPassesAVarToShowTheCorrelationThatSoundsGoodRightRight = ...`